data-manipulation/hashing/md5/hash-data-with-md5.yml

```diff
@@ -32,6 +32,5 @@ rule:
           - number: 0x8003 = CALG_MD5
           - api: advapi32.CryptCreateHash
       - and:
-        - format: dotnet
```

Comment:
need to find a good solution for this, see https://github.com/mandiant/capa/discussions/1106#discussioncomment-3202916 and #601